import_email_events.py

```diff
@@ -0,0 +1,24 @@
+#
+# Script to import "email event" metrics from S3 into redshift.
+#
+
+import import_events
+SCHEMA = """
+    type VARCHAR(30) NOT NULL ENCODE lzo,
+    locale VARCHAR(40) ENCODE lzo,
+    domain VARCHAR(40) ENCODE lzo,
+    template VARCHAR(64) ENCODE lzo,
+    flow_id VARCHAR(64) DISTKEY ENCODE lzo,
+    complaint VARCHAR(64) ENCODE lzo,
+    bounced VARCHAR(64) ENCODE lzo
+"""
+
+COLUMNS = "type, locale, domain, template, flow_id, complaint, bounced"
+
+import_events.run(s3_prefix="fxa-email/data/email-events",
+                  event_type="email",
+                  temp_schema=SCHEMA,
+                  temp_columns=COLUMNS,
+                  perm_schema=SCHEMA,
+                  perm_columns=COLUMNS)
```

Comment:
(also, I think you've already done this locally but, as per IRC, we'll need to add `id_column="flow_id"` to these arguments)